### ISSUE TITLE
Question on code of zOrder() Method.

### ISSUE THREAD

**Issue by thierry810** [earcut4j/earcut4j#5]
Question on code of zOrder() Method.

**Comment by thierry810** [comment]
Reading the code of zOrder method, at line 284, i am wondering if it should not be 
`  ly = (ly | (ly << 8)) & 0x00FF00FF; `
instead of  
`  y = (ly | (ly << 8)) & 0x00FF00FF;`
and so on for the three next lines.

Cordially Thierry

**Comment by jsimomaa** [comment]
Good catch and you are absolutely correct! I committed a fix to this, thanks
